azure-pipelines.yml

```
@@ -307,6 +307,26 @@ stages:
         inputs:
           artifact: DotnetTestResultsNet6
           targetPath: 'TestResults'
+          
+      - powershell: |
+          # In order to speed up our build we chose to run the tests and compute the code coverage in parallel.
+          # This might result in different paths in the coverage reports, and in order to be able to properly import
+          # the coverage data we need to normalize the path.
+          # e.g.
+          #   the machine that computes the coverage has the base path `C:\ProgramData\vsts-agent\1\s`
+          #   and the one that imports it `C:\ProgramData\vsts-agent\2\s`
```

I'd update it to something like this.
```suggestion
          # Coverage is computed in another job and can have a different base path. We need to normalize content of the coverage files to the current job path. Values generated by the agent can look like this:
          #   `C:\ProgramData\vsts-agent\1\s`
          #   `C:\ProgramData\vsts-agent\2\s`
```